Breeze.UI/src/app/wallet/tumblebit/tumblebit.component.ts

```diff
@@ -217,10 +217,11 @@ export class TumblebitComponent implements OnDestroy {
             this.genericModalService.openModal(
               Error.toDialogOptions('Failed to get general wallet information. Reason: API is not responding or timing out.', null));
           } else if (error.status >= 400) {
-            if (!error.json().errors[0]) {
+            const firstError = error.json().errors[0];
```

Comment:
You could use getFirstError from Error helper class https://github.com/BreezeHub/BreezeProject/blob/master/Breeze.UI/src/app/shared/classes/error.ts